content/accessibility/alternative-text-for-images.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: Alternative Text for Images
```

Comment:
```suggestion
title: Alternative text for images
```


I only know this because it was called out in my PRs 😅 [Sentence case is preferred for titles](https://primer.style/contribute/documentation#grammar-and-usage)